spec.md

```diff
@@ -41,24 +39,37 @@ secPrefix:
   - Sections
 # This 'lastDelim' option does not work...
 lastDelim: ", and"
+# Information about revision history.  This is used by the LaTeX template.
+revisionHistory:
+  # Information about what was added in the current version.  This will be
+  # populated using the "version" that the Makefile grabs from git.  Notable
+  # additions to the specification should append entries here.
```

Comment:
ok so the idea is that PR authors should add something to this section for not-NFC changes? SGTM

Reply:
Yep! We can also move this into a separate `revisionHistory.yaml` file to make it more obvious (and get better support for YAML highlighting/editor modes than the inline stuff supports).